tools/sigclient/pkg/alerts/alerts_loadtest.go

```diff
@@ -265,13 +266,11 @@ func trackNotifications(webhookChan chan alertutils.WebhookBody, numAlerts int,
 				if minuteData, exists := summary.MinuteData[minute]; exists {
 					minuteData.AverageDelay = minuteData.TotalDelay / time.Duration(minuteData.TotalAlertsReceived)
 					logMinuteSummary(minute, minuteData, numAlerts)
-					summary.MinuteData[minute] = minuteData
+					delete(summary.MinuteData, minute)
 				}
 			}
 		case <-timeout.C:
-			log.Errorf("Timed out waiting for alerts")
-			successChan <- false
-			return
+			log.Errorf("%vCurrentMinute=%v,Timeout=%v,ErrorMessage=Timed out waiting for alerts", LOG_MSG_DELIM, currentMinute, true)
 		case <-exitChan:
```

Comment:
Do you want to also reset the timeout timer here? Or just log the error?

Reply:
Yeah that's a good Point. But I have actually executed and checked the scenario where it would throw this timeout and then I let the SigLens Server send out Alerts and then the Alerts started showing and the timeout got reset. So I stopped the SigLens server and then again the timeout error logs printed after two minutes.

So I think this is fine.

![image](https://github.com/siglens/siglens/assets/29257812/ea81a14b-78f0-4738-93f6-bd148fd52892)